vocabularies/UI.xml

```diff
@@ -1389,6 +1390,55 @@ The trend is
         </Member>
       </EnumType>
 
+      <Term Name="Note" Type="UI.NoteType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated entity type can be visualized in a reuse component for notes" />
+        <Annotation Term="Core.LongDescription">
+          <String>Besides this annotation the reuse component also evaluates the annotations
+          [`Common.CreatedBy`](Common.md#CreatedBy),
+          [`Common.CreatedAt`](Common.md#CreatedAt),
+          [`Common.ChangedBy`](Common.md#ChangedBy),
+          [`Common.ChangedAt`](Common.md#ChangedAt)
+          on the same entity type.</String>
+        </Annotation>
+      </Term>
+      <ComplexType Name="NoteType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="Title" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Title of the note" />
+        </Property>
+        <Property Name="Content" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Content of the note, as a string" />
+          <Annotation Term="Core.LongDescription">
+            <String>The reuse component also evaluates the [`Core.MediaType`](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.md#MediaType)
+            annotation on this property.</String>
+          </Annotation>
+        </Property>
+        <Property Name="Type" Type="Edm.String" Nullable="false">
```

Comment:
What is the type, can you pls. mention an example?

Reply:
I described it here with and screenshot of the UI https://github.com/SAP/odata-vocabularies/pull/221#issuecomment-1412315655 and also here https://github.com/SAP/odata-vocabularies/pull/221#discussion_r1115311905

Reply:
Thanks, @thomas-kaltenbach . Where does this type come from - the screenshot shows a dropdown. So there must a "ValueList EntitySet" exist that contains these value. And is this something the app defines, or the customer, or is it overarching (a central team defines it)?

Reply:
The application defines a notetype and assign it to applications SAPObjectNodeType. For ABAP Cloud we are also developing ADT UI for these entities (NTTY/NTTA). 
So we know which notetypes can be expected and also provide the create button for these notetypes.
Therefore we have the entityset currently called `I_NoteBasicType`.